modules/luci-base/htdocs/luci-static/resources/fs.js

```diff
@@ -342,7 +342,7 @@ var FileSystem = L.Class.extend(/** @lends LuCI.fs.prototype */ {
 		var postdata = 'sessionid=%s&path=%s'
 			.format(encodeURIComponent(L.env.sessionid), encodeURIComponent(path));
 
-		return L.Request.post('/cgi-bin/cgi-download', postdata, {
+		return L.Request.post(L.env.cgi_base + 'cgi-download', postdata, {
```

Comment:
Can you add an explicit trailing slash here and in the other places?

`L.env.cgi-base + '/cgi-download'`